tasks/copy.js

```diff
@@ -92,17 +93,17 @@ gulp.task('copy:fonts', () => {
  */
 gulp.task('copy:assets', callback => {
 
-    const getPackage = path => {
-        const split = path.split('/'); // e.g. [...'@justeat', '', 'fozzie', '']
+    const getPackage = filepath => {
+        const split = filepath.split('/'); // e.g. [...'@justeat', '', 'fozzie', '']
         return {
-            path,
+            filepath,
             name: split[split.length - 2]
         };
     };
 
     const copyFromPackage = pkg => new Promise((resolve, reject) => {
-        gutil.log(`❯❯ Copying any assets in ${pkg.path} to ${pathBuilder.importedAssetsDistDir}/${pkg.name}`);
-        copyAssets(pkg.path, `${pathBuilder.importedAssetsDistDir}/${pkg.name}`, err => {
+        gutil.log(`❯❯ Copying any assets from ${pkg.name} to ${path.dirname(config.assetDistDir)}`);
+        copyAssets(`${pkg.filepath}`, `${path.dirname(config.assetDistDir)}`, err => {
```

Comment:
I'm using path.dirname here to navigate up one directory from `dist/` so that `copyAssets` merges the current `dist/` with the one from the package. This also covers the use of `config.assetDistDir = 'assets/dist/` as seen in globalweb and the docs site.